### ISSUE TITLE
Opus file not getting played/convert

### ISSUE THREAD

**Issue by swativishnoi** [rowntreerob/Google-speech-Opus-Recorder#2]
We are getting Opus file generated while recording. But we tried playing it and also converting it to mp3 using online converter(http://audio.online-convert.com/convert-to-mp3). So could you please help us by just helping on this opus file format.

**Comment by rowntreerob** [comment]
will look into this issue when feasible... in meantime you might compare the pastebin logcat to your own logcat, looking for any obvious bugs/ discrepancy between the 2.      https://pastebin.com/LRfRiMA0

**Comment by rowntreerob** [comment]
bug confirmed.  in the orig. project i only tested for the creation of the output and never played it back!  

I created the opus file and tried to play it using 'ffplay' .. It looks like legitimate mimetype opus container but no sound plays. I looked at the source project "https://github.com/louisyonge/opus_android"  and will have to do more digging on why outputs are not producing any sound.  sorry may take awhile to complete the debug i should have done earlier...
